```
/app/webkit/Source/WebCore/platform/graphics/gstreamer/mse/MediaSourcePrivateGStreamer.cpp: In member function ‘WebCore::TrackID WebCore::MediaSourcePrivateGStreamer::registerTrackId(WebCore::TrackID)’:
/app/webkit/Source/WebCore/platform/graphics/gstreamer/mse/MediaSourcePrivateGStreamer.cpp:235:10: error: variable ‘result’ set but not used [-Werror=unused-but-set-variable]
  235 |     auto result = m_trackIdRegistry.add(assignedId);
      |          ^~~~~~
```

Right, have to do the maybe_unused thing...

--- a/Source/WebCore/platform/graphics/gstreamer/mse/MediaSourcePrivateGStreamer.cpp
+++ b/Source/WebCore/platform/graphics/gstreamer/mse/MediaSourcePrivateGStreamer.cpp
@@ -232,7 +232,8 @@ TrackID MediaSourcePrivateGStreamer::registerTrackId(TrackID preferredId)
     auto maxRegisteredId = std::max_element(m_trackIdRegistry.begin(), m_trackIdRegistry.end());
     auto assignedId = std::max((TrackID) 100, *maxRegisteredId + 1);
 
-    ASSERT(m_trackIdRegistry.add(assignedId).isNewEntry);
+    [[maybe_unused]] auto result = m_trackIdRegistry.add(assignedId);
+    ASSERT(result.isNewEntry);
     if (player)
         GST_DEBUG_OBJECT(player->pipeline(), "Registered new Track ID: %" PRIu64 " (preferred ID would have been %" PRIu64 ")", assignedId, preferredId);
 

